e2e-app/src/app/modal/autoclose/modal-autoclose.po.ts

```diff
@@ -1,26 +1,26 @@
-import {test} from '../../../../playwright.conf';
+import {getPage} from '../../../../baseTest';
 import {timeoutMessage} from '../../tools.po';
 import {waitForModalCount} from '../modal.po';
 
 export const clickOnReset = async() => {
-  await test.page.click('#reset-button');
+  await getPage().click('#reset-button');
 };
 
 export const clickOnClose = async() => {
-  await test.page.click('#modal-close-button');
+  await getPage().click('#modal-close-button');
   await waitForModalCount(0);
 };
 
 export const openModal = async(option = '') => {
   if (option !== '') {
-    await test.page.click(`#option-${option}`);
+    await getPage().click(`#option-${option}`);
   }
 
-  await test.page.click('#open-modal');
+  await getPage().click('#open-modal');
   await waitForModalCount(1);
 };
 
 export const waitDismissReason = async(expected, error) => {
   await timeoutMessage(
```

Comment:
Probably timeoutMessage is not needed anymore (everywhere), as we can use the expect features for that purpose.

For example here:

```typescript
await expect(getPage().locator('#dismiss-reason'), error).toHaveText(expected);
```